setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name='evergreen.py',
-    version='0.6.1',
+    version='0.6.2',
```

Comment:
This looks like it breaks backwards compatibility. So it should probably bump the version to 0.7.0.

Reply:
Breaking change would push this to `1.0.0`, no? Was trying to be sneaky and just remove the broken API since logically no one could have integrated it yet by virtue of being broken 😄

Reply:
Looking at semvar, I guess it doesn't matter until we hit `1.0.0`.